src/middleware/authorizeAdmin.js

```diff
@@ -0,0 +1,15 @@
+const errorMessages = require('../constants/errorMessages');
+
+const authorizeAdmin = (req, res, next) => {
+  if (!req.user) {
+    return res.status(401).json({ error: errorMessages.UNAUTHORIZED });
+  }
+
+  if (req.user.role !== 'admin') {
+    return res.status(403).json({ error: errorMessages.FORBIDDEN });
+  }
```

Comment:
instead of hardcoded values for status code use the library 

Also 'admin' should be inside constant